source/community/activities/outreachy.html.md

```diff
@@ -44,6 +44,8 @@ Please find the mentors and projects listed below. In case you are interested in
 
 **Contact Info:** mail: mzamazal@redhat.com, irc: mzamazal in #ovirt channel in irc.oftc.net
 
+I'm sorry, I no longer accept new candidates.
```

Comment:
You mean adding something like "you can still chose from the other oVirt projects"?